artifacts-download-trigger/action.yml

```diff
@@ -0,0 +1,95 @@
+name: Workflow Dispatch and wait to download artifact
```

Comment:
I am really not sure if we need all the inputs.
It is a really blowed up inputs list for our purpose, isn't it?

Reply:
Better have them then need them!